packages/BetterPhpDocParser/Printer/PhpDocInfoPrinter.php

```diff
@@ -55,7 +55,7 @@ final class PhpDocInfoPrinter
     /**
      * @var string
      */
-    private const NEWLINE_WITH_ASTERISK = PHP_EOL . ' * ';
+    private const NEWLINE_WITH_ASTERISK = "\n" . ' * ';
```

Comment:
I had 2 options in mind to fix this:
A) keep `PHP_EOL` and normalize the newlines - requires some newline style detections etc.
B) simpliy just use `"\n"` since most of source code uses `\n` - even on windows.

I went for A) since it was very simple and should work. B) would be way more involved.

Reply:
Thanks :+1: 

Could you add comment why the constant is not used? That would revert by somebody not using Windows in the future

Reply:
Done